project.php

```diff
@@ -37,22 +45,28 @@
 <header class="p-3 bg-teal-50 flex justify-between">
     <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
     <div class="pr-3 flex">
-        <a href=<?php echo "project.php?project=$idLink"?> class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+        <?php
+        if ($pageLocale == 'uk') {
+            echo "<a href = 'project.php?project=$idLink&location=uk' class='p-3 bg-slate-300 rounded-l-lg border-y border-l' > 🇬🇧</a >";
+            echo "<a href = 'project.php?project=$idLink&location=us' class='p-3 rounded-r-lg border-y border-r' > 🇺🇸</a >";
+        } else {
+            echo "<a href = 'project.php?project=$idLink&location=uk' class='p-3  rounded-l-lg border-y border-l' > 🇬🇧</a >";
+            echo "<a href = 'project.php?project=$idLink&location=us' class='p-3 bg-slate-300 rounded-r-lg border-y border-r' > 🇺🇸</a >";
+        }
```

Comment:
There's a smarter way of doing this: it seems that the only difference is that the `bg-slate-300` class is set for the current locale, otherwise it's not there at all. So you can just echo both lines with inline ternarys or using `&&` to only add `bg-slate-300` into the class list string if the locale matches the link. Does that make sense?